src/System.Text.Encodings.Web/src/Configurations.props

```diff
@@ -1,8 +1,10 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.0;
+      netstandard1.1;
```

Comment:
The netstandard1.0 asset will be harvested now, which is probably fine... but why introduce a netstandard1.1 target?

Reply:
I'll be using `Span<T>` and `ROS<T>` in the implementation to avoid some of the unsafe code, and the `System.Memory` package requires netstandard 1.1+.

Reply:
Sure... but why bother with another version between 1.0 and 2.0.  The harvested 1.0, a refreshed 2.0, and the improved 2.1 seem sufficient to me...

Reply:
Ok, can kill it. :)